File: micro_bit/jazda-po-kwadracie-2/main.py
def on_button_pressed_a():
    for index in range(4):
        AlphaBot2.run(Dir.FORWARD, 30)
        basic.pause(1000)
        AlphaBot2.motor_run(Motors.M1, -30)
        basic.pause(800)
    AlphaBot2.run(Dir.STOP, 0)
input.on_button_pressed(Button.A, on_button_pressed_a)

def on_forever():
    pass
basic.forever(on_forever)

File: micro_bit/jazda-po-kwadracie-2/main.ts
input.onButtonPressed(Button.A, function () {
    for (let index = 0; index < 4; index++) {
        AlphaBot2.Run(Dir.forward, 30)
        basic.pause(1000)
        AlphaBot2.MotorRun(Motors.M1, -30)
        basic.pause(800)
    }
    AlphaBot2.Run(Dir.stop, 0)
})
basic.forever(function () {
	
})
